ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/StandardComponentInitializer.java

```diff
@@ -649,6 +649,18 @@ public void initialize() {
         actionManager.registerAction("noOpAction", new NoOpAction());
         actionManager.registerAction("signatureHelp", signatureHelpAction);
 
+        DefaultActionGroup editorContext = new DefaultActionGroup(actionManager);
```

Comment:
Variable name **editorContext** doesn't describe the data it contains.
IMO **editorContextMenu** or **editorContextMenuGroup** would be much more better.
WDYT?